tests/integration_tests/test_examples.py

```diff
@@ -304,3 +306,27 @@ def test_rl2_ml10_meta_test():
         '--episode_per_task', '1', '--meta_batch_size', '10'
     ],
                           check=False).returncode == 0
+
+
+@pytest.mark.mujoco
+@pytest.mark.no_cover
+@pytest.mark.timeout(200)
```

Comment:
Is this timeout the reason you've written a test case manually (instead of just using the default example tester)?
If so, can you write a comment explaining why (and for the test below).

Reply:
fixed